api/v1/Makefile

```diff
@@ -51,4 +51,4 @@ clean:
 
 .PHONY: deps
 deps:
-	@$(PROTOC) --version | grep 3.11.4 || (echo "Error: protoc 3.11.4 is required" && exit 1)
```

Comment:
i wonder if there is a better way to vendor this in, so it's per-project, not system-wide

Reply:
I believe that @gandro wanted to update the Makefile to use a docker container instead of the host binary at some point.
I guess this would do.

Reply:
Yeah that works for me. No action needed on this PR